lib/ood_core/job/adapters/lsf.rb

```diff
@@ -0,0 +1,200 @@
+require "ood_core/refinements/hash_extensions"
+
+module OodCore
+  module Job
+    class Factory
+      using Refinements::HashExtensions
+
+      # Build the Lsf adapter from a configuration
+      # @param config [#to_h] the configuration for job adapter
```

Comment:
Not yet. After it works I'm going to send an email to George with a list of questions about LSF. There is a `-m` argument for the cluster, but when I provide the name of the cluster as returned by the LSF command to get a list of available clusters the commands don't run.